tests/lib/rules/require-valid-default-prop.js

```diff
@@ -252,6 +252,28 @@ ruleTester.run('require-valid-default-prop', rule, {
       parserOptions: { ecmaVersion: 6, sourceType: 'module' },
       parser: require.resolve('@typescript-eslint/parser'),
       errors: errorMessage('function')
+    },
+    {
+      filename: 'test.vue',
```

Comment:
Maybe add a reference to https://github.com/vuejs/eslint-plugin-vue/issues/1853 here:

```suggestion
      // https://github.com/vuejs/eslint-plugin-vue/issues/1853
      filename: 'test.vue',
```